atd-toolbox/direction_movement/app/view_with_stakeholder_feedback_criteria.sql

```diff
@@ -0,0 +1,29 @@
+CREATE OR
+REPLACE VIEW check_vz_restore AS (
+    SELECT
+        movement_direction_corrections. *,
+        crashes.qa_status,
+        crashes.position,
+        concat(
+            'update atd_txdot_units set ',
+            field,
+            ' = ',
+            value,
+            ' where unit_id = ',
+            unit_id,
+            ';'
+        ) as update_statement
```

Comment:
The values out of this column are copied into a scratch file and then piped back into the database to execute the restoration.